src/components/SignInForm/index.jsx

```diff
@@ -80,24 +68,26 @@ function SignIn() {
             onChange={handleChange}
           />
           <Button name="signIn" text="Sign In" />
-          {signInIsError && <Error>{signInError.data.error}</Error>}
+          {signInIsError && (
```

Comment:
Maybe you can also hide the error if the request is loading:


```js
{!signInLoading && signInIsError && (
...
```

I think it could improve the UX for the user a bit when they attempt to resubmit a form that already had an error from a previous submit.